### PR TITLE
R93

### PR DESCRIPTION
**Pull request by rzats**

### CODE REVIEW

**Comment by alexanderdean** [comment]
Please tidy up the commit history (but without losing the attribution of the original developers)

**Comment by alexanderdean** [comment]
Sorry didn't mean to close!

File: jsonpaths/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment_1.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "$.data.parameters.excludeUseragentFile.uri",
+        "$.data.parameters.includeUseragentFile.database",
+        "$.data.parameters.includeUseragentFile.uri",
+        "$.data.parameters.ipFile.database",
+        "$.data.parameters.ipFile.uri",
+        "$.data.vendor",
+        "$.data.parameters.httpPassword",
+        "$.data.parameters.httpUsername"
+    ]
+}
```
Comment (BenFradet) [inline jsonpaths/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment_1.json]: I don't think this and the associated sql are needed btw

Comment (alexanderdean) [reply jsonpaths/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment_1.json]: Agree

**Comment by rzats** [comment]
@BenFradet igluctl artifacts are fixed now

**Comment by alexanderdean** [comment]
Missing the Prepared for release commit

**Comment by alexanderdean** [comment]
@misterpig @miike - this is shaping up as the next Iglu Central release. Please shout if you see anything unexpected.

**Comment by alexanderdean** [comment]
Can we get the green ticket from you @BenFradet ?

File: schemas/com.iab.snowplow/spiders_and_robots/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for IAB Robots&Spiders POJO",
```
[the remark this answers is not in the document]
Comment (alexanderdean) [reply schemas/com.iab.snowplow/spiders_and_robots/jsonschema/1-0-0]: Good call @chuwy - @rzats please make that change and then I will green tick and release

[the remark this answers is not in the document]
Comment (rzats) [reply schemas/com.iab.snowplow/spiders_and_robots/jsonschema/1-0-0]: @chuwy @alexanderdean this is fixed now

File: schemas/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for IAB spiders&robots enrichment",
```
Comment (chuwy) [inline schemas/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment/jsonschema/1-0-0]: @rzats could you please normalize this one as well (capitalize, white spaces)

Comment (rzats) [reply schemas/com.snowplowanalytics.snowplow.enrichments/iab_spiders_and_robots_enrichment/jsonschema/1-0-0]: fixed!

File: schemas/com.iab.snowplow/spiders_and_robots/jsonschema/1-0-0 [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for IAB Spiders & Robots POJO",
```
Comment (alexanderdean) [inline schemas/com.iab.snowplow/spiders_and_robots/jsonschema/1-0-0]: Can we call this data, not "POJO". POJO is unnecessary Javaworld-bleedthrough

**Pull request merged**